opy/_regtest/src/core/state.py

```diff
@@ -78,6 +78,7 @@ def Disable(self):
     ('x', 'xtrace'),
     ('f', 'noglob'),
     ('C', 'noclobber'),
+    ('E', 'errtrace'),
```

Comment:
Oh anything in `opy/_regtest` does not need to be changed

This is a test file that's not active

Please revert this -- thanks for searching